packages/manager/src/features/Account/utils.ts

```diff
@@ -0,0 +1,77 @@
+import { getStorage, setStorage } from 'src/utilities/storage';
+
+import type { Token } from '@linode/api-v4';
+
+// TODO: Parent/Child: FOR MSW ONLY, REMOVE WHEN API IS READY
+// ================================================================
+// const mockExpiredTime =
+//   'Mon Nov 20 2023 22:50:52 GMT-0800 (Pacific Standard Time)';
+// ================================================================
+
+/**
+ * Determine whether the tokens used for switchable accounts are still valid.
+ */
+export const isParentTokenValid = ({
+  isProxyUser,
+}: {
+  isProxyUser: boolean;
+}) => {
+  const now = new Date().toISOString();
+
+  // From a proxy user, check whether parent token is still valid before switching.
+  if (
+    isProxyUser &&
+    now >
+      new Date(getStorage('authentication/parent_token/expire')).toISOString()
+
+    // TODO: Parent/Child: FOR MSW ONLY, REMOVE WHEN API IS READY
+    // ================================================================
+    // new Date(mockExpiredTime).toISOString()
+    // ================================================================
+  ) {
+    return false;
+  }
+  return true;
+};
+
+/**
+ * Set token information in the local storage.
```

Comment:
```suggestion
 * Set token information in the local storage.
 * This allows us to store a token for later use, such as switching between parent and proxy accounts.
```

I think we should document this more clearly.